design/ErrorHandling.md

```diff
@@ -157,9 +297,6 @@ It is only necessary if an application wishes to recover from recoverable errors
 
 ## Open Questions and Considerations
 
- - WebGPU could guarantee that objects such as `GPUQueue` and `GPUFence` can never be errors.
```

Comment:
Why is is deleted in this PR?

Reply:
Oops, splitting error. This is meant to be in 215.